osquery/core/conversions.cpp

```diff
@@ -316,49 +315,44 @@ bool JSON::valueToBool(const rj::Value& value) {
   return false;
 }
 
-std::string base64Decode(const std::string& encoded) {
-  std::string is;
-  std::stringstream os;
-
-  is = encoded;
-  boost::replace_all(is, "\r\n", "");
-  boost::replace_all(is, "\n", "");
-  size_t size = is.size();
+std::string base64Decode(std::string encoded) {
+  boost::replace_all(encoded, "\r\n", "");
+  boost::replace_all(encoded, "\n", "");
 
   // Remove the padding characters
-  if (size && is[size - 1] == '=') {
-    --size;
-    if (size && is[size - 1] == '=') {
-      --size;
+  for (int i = 0; i < 2; i++) {
+    if (!encoded.empty() && encoded.back() == '=') {
+      encoded.pop_back();
     }
   }
 
-  if (size == 0) {
+  if (encoded.empty()) {
     return "";
```

Comment:
Instead of creating new string to return from C-string, you can just return `encoded`, it is already empty.